docs/contributing/docs-templates.md

```diff
@@ -21,7 +22,19 @@ Here are templates (models) to follow when contributing to Gatsby docs to ensure
 
 Reference guide articles cover discrete topics as documentation with links to other resources. A reference guide explains a Gatsby concept or technique without the step-by-step context provided by a tutorial or recipe.
 
-Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios.
+Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios. Reference material would list text headings for each section of a guide, rather than "steps 1-5" like a tutorial.
+
+#### Other types of guide articles
+
+The Gatsby docs include multiple types of guides. They mostly follow the same format, but some docs will have a slightly different purpose and/or audience.
+
+- [Reference Guides](/docs/guides/): explanations of Gatsby site development techniques and common workflows. These should be written for all skill-levels.
+- [API Reference](/docs/api-reference/): technical docs on API methods and options, common files, and links to additional resources and explanations.
+- [Releases & Migration](/docs/releases-and-migration/): release notes and guides on how to upgrade Gatsby and third-party packages.
+- [Conceptual Guide](/docs/conceptual-guide/): high-level docs to explain important Gatsby concepts and philosophies.
```

Comment:
```suggestion
- [Conceptual Guides](/docs/conceptual-guide/): high-level docs to explain important Gatsby concepts and philosophies.
```

Reply:
I went back and forth on this, and decided to go with what was in the URL/section headings. But I don't think it's a big deal to pluralize it here since that's how I'd talk about it in conversation.